### PR TITLE
feat: Add mana details to Output page

### PR DESCRIPTION
**Pull request by msarcev**
# Description of change

- Added fields displaying stored, stored with decay, potential and total mana of the output, in the Metadata section of Output page

Closes https://github.com/iotaledger/explorer/issues/999
Closes https://github.com/iotaledger/explorer/issues/1000

## Type of change

- Enhancement (a non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/app/routes/nova/OutputPage.tsx [hunk 1]
```diff
@@ -47,6 +50,20 @@ const OutputPage: React.FC<RouteComponentProps<OutputPageProps>> = ({
 
     const { blockId, transactionId, outputIndex, isSpent, transactionIdSpent } = outputMetadataResponse ?? {};
 
+    // @ts-expect-error TODO: Remove this ignore once included/spent are honoured in the type https://github.com/iotaledger/iota-sdk/issues/1884
+    const createdSlotIndex = (outputMetadataResponse?.included?.slot as number) ?? null;
+    // @ts-expect-error TODO: Remove this ignore once included/spent are honoured in the type https://github.com/iotaledger/iota-sdk/issues/1884
+    const spentSlotIndex = (outputMetadataResponse?.spent?.slot as number) ?? null;
+
+    let outputManaDetails: OutputManaDetails | null = null;
+    if (output !== null && createdSlotIndex !== null && protocolInfo !== null) {
```
Comment (brancoder) [inline client/src/app/routes/nova/OutputPage.tsx:59]: we could simplify these checks with just `if (output && createdSlotIndex && protocolInfo)`
and also inside the if clause we could on only figure out the `slotIndex` and call `buildManaDetailsForOutputs` only once
along the line of:
```
if (output && createdSlotIndex && protocolInfo) {
    const slotIndex = isSpent && spentSlotIndex ? 
    spentSlotIndex :
    latestConfirmedSlot > 0 ? latestConfirmedSlot : null;
    outputManaDetails = slotIndex ? buildManaDetailsForOutput(output, createdSlotIndex, slotIndex, protocolInfo.parameters) : null;
}
```

File: client/src/app/routes/nova/OutputPage.tsx [hunk 2]
```diff
@@ -47,6 +50,20 @@ const OutputPage: React.FC<RouteComponentProps<OutputPageProps>> = ({
 
     const { blockId, transactionId, outputIndex, isSpent, transactionIdSpent } = outputMetadataResponse ?? {};
 
+    // @ts-expect-error TODO: Remove this ignore once included/spent are honoured in the type https://github.com/iotaledger/iota-sdk/issues/1884
```
Comment (brancoder) [inline client/src/app/routes/nova/OutputPage.tsx:53]: this issue got resolved so we could remove these comments: https://github.com/iotaledger/iota-sdk/issues/1884

**Review by brancoder — CHANGES_REQUESTED** [review]

File: client/src/app/routes/nova/OutputPage.tsx (hunk 1 above)
Comment (msarcev) [reply client/src/app/routes/nova/OutputPage.tsx:62]: Updated this part a bit. But `slotIndex` cannot be allowed to be `null` as in the snippet above.

Comment (brancoder) [reply client/src/app/routes/nova/OutputPage.tsx:62]: In proposed snippet there is a check for slot index not being null `slotIndex ?` and only the do we call `buildManaDetailsForOutput` :thinking:

Comment (msarcev) [reply client/src/app/routes/nova/OutputPage.tsx]: My bad, overlooked the last ternary.

**Pull request merged**